New package: HEtools v1.0.0-DEV

JuliaRegistrator opened the pull request:
- Registering package: HEtools
- Repository: https://github.com/bquast/HEtools.jl
- Created by: @bquast
- Version: v1.0.0-DEV
- Commit: 191940b3b3165b2ece020ca60302e860ef940360
- Git reference: HEAD

goerz commented:
This package appears to be basically empty at this point. Please see the guidelines for [Should I register my package now?](https://github.com/JuliaRegistries/General/blob/master/README.md#should-i-register-my-package-now)

bquast commented:
Thank you @goerz I will add the other functions before I upload it.

bquast commented:
Not sure what is going wrong, will look at it in  few days

JuliaHubRegistrator committed:
New package: HEtools v1.0.0-DEV

UUID: ac54a25c-9fbc-4d8a-9af3-833e06b1b3a5
Repo: https://github.com/bquast/HEtools.jl.git
Tree: ac4211b2aa627c99ec51e72c130cbbae3982a1d7

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09

bquast commented:
@goerz I have the package ready to add the other functions, but it seems impossible to get the checks to pass.

The dirty registry error keeps showing up. I've switched from macports Julia to the pre-compiled binary from the website.

I've rebuilt the (nearly empty) package from Scratch. Switched to different dependency notation.

None of it works, please advise how I can get a package to past the tests. 

Sorry just doing this Julia development for the first time, but i have a bunch of packages in FHE and ML that I want to translate to Julia.
